CompatBot/Commands/Misc.cs

```diff
@@ -63,15 +66,16 @@ internal sealed class Misc: BaseCommandModuleCustom
             "So cute!", "It's so nice, I think about it every day!", "😊 Never expected to be this pretty!", "It's overflowing with charm!", "Filled with passion!",
             "A love magnet", "Pretty Fancy", "Admirable", "Sweet as a candy", "Delightful",
             "Enchanting as the Sunset", "A beacon of hope!", "Filled with hope!", "Shiny!", "Absolute Hope!",
-            "The meaning of hope", "Inspiring!",
+            "The meaning of hope", "Inspiring!", "Oooh, you're a smart cookie", "I bet you sweat glitter.", "Better than bubble wrap.",
```

Comment:
The smart cookie doesn’t really give any answer, so should be in neutral I think. Samw with glitter.